contracts/BC_fusion/BSCGovernor.sol

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorSettingsUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/governance/GovernorUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/governance/compatibility/GovernorCompatibilityBravoUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorVotesUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorTimelockControlUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorVotesQuorumFractionUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorPreventLateQuorumUpgradeable.sol";
+
+
+contract BSCGovernor is
+    GovernorUpgradeable,
+    GovernorSettingsUpgradeable,
+    GovernorCompatibilityBravoUpgradeable,
+    GovernorVotesUpgradeable,
+    GovernorTimelockControlUpgradeable,
+    GovernorVotesQuorumFractionUpgradeable,
+    GovernorPreventLateQuorumUpgradeable
+{
+    constructor() {
+        _disableInitializers();
+    }
+
+    function initialize(
+        IVotesUpgradeable _token,
+        TimelockControllerUpgradeable _timelock,
+        address _owner,
+        uint256 _votingDelay,
+        uint256 _votingPeriod,
+        uint256 _quorumNumerator,
+        uint256 _proposalThreshold,
+        uint64 _minPeriodAfterQuorum
+    ) external initializer {
+        __Governor_init("BSCGovernor");
+        __GovernorSettings_init(_votingDelay, _votingPeriod, _proposalThreshold);
+        __GovernorCompatibilityBravo_init();
+        __GovernorVotes_init(_token);
+        __GovernorTimelockControl_init(_timelock);
+        __GovernorVotesQuorumFraction_init(_quorumNumerator);
+        __GovernorPreventLateQuorum_init(_minPeriodAfterQuorum);
+    }
+
+    function state(
+        uint256 proposalId
+    ) public view override(GovernorUpgradeable, IGovernorUpgradeable, GovernorTimelockControlUpgradeable) returns (ProposalState) {
+        return super.state(proposalId);
+    }
+
+    function propose(
```

Comment:
I think only need to override `GovernorCompatibilityBravoUpgradeable`, please check other function as well.

Reply:
In the case of multiple inheritance, you must explicitly override all father contracts since `GovernorCompatibilityBravoUpgradeable` is also inherited from `GovernorUpgradeable`